webpack.config.js

```diff
@@ -6,7 +6,7 @@ const isProduction = process.env.NODE_ENV == 'production';
 const HtmlBundlerPlugin = require('html-bundler-webpack-plugin');
 
 const config = {
-  entry: './client/src/index.js',
```

Comment:
Using the Bundler Plugin, the entry point is a HTML template, not JavaScript.
All source files of script and styles should be specified directly in HTML.